docs/user-doc.md

```diff
@@ -0,0 +1,33 @@
+# User documentation
+
+This app is part of the 
+[PCIC Data Portal](https://github.com/pacificclimate/pdp), 
+and as such its user documentation
+is integrated within the larger set of PDP documentation.
+
+This app's user documentation is maintained _in this repo_ and is copied
+over (from GitHub) each time the PDP documentation is built. This allows
+the user documentation to be maintained in lockstep with code updates
+in this repo.
+
+Note: The PDP is a Python project; therefore its documentation is coded in
+rST (reStructuredText); therefore the documentation for this portal
+must also be coded in rST. This is a bit foreign to a JS project, but
+it gives an excellent result.
+
+All user documentation is contained in the directory 
+[`user-doc`](../user-doc) in this repo.
```

Comment:
Admittedly, I was initially a little confused about why the `user-doc.md` file appeared to be communicating to the developer. Would it be worth reorganizing the docs into `docs/developer/...` and `docs/user/...` directory hierarchy?

Reply:
Yes